### DISCUSSION TITLE
Logos: Colors! [Deadline: March 8]

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#95] [Consensus Building]
[![🗳️ Vote progress](https://www.commonhaus.org/votes/commonhaus/foundation-draft/95.svg)](https://github.com/commonhaus/foundation-draft/discussions/95#discussioncomment-8726122 "DC_kwDOKRPTI84AhSZq")

### Proposal

Finally, we can choose some colors!

> [!IMPORTANT]
> View the details: [Round 4 colors](https://docs.google.com/presentation/d/1TVPc9zAD71uyxl6Zb6w5bdhb9fQKaYNKOxA1PXbH2UE/edit?usp=sharing)  
> There is more going on than just the logo.

<img width="90%" alt="comparison of logo with colors" src="https://github.com/commonhaus/foundation-draft/assets/808713/dad82306-a572-4a52-a4b7-48cb426f3ae7">

### Voting group

@commonhaus/founders

### Consensus mechanism

Rank them in order of preference.

- (🚀, optional) Specific number if you have a favorite
- (👎, optional) Specific number if it is an absolute no

Constructive commentary is helpful.

If you have other suggestions, try to put together a full palette to demonstrate.

**Comment by aalmiray** [comment]
4, 3, 1.

3 & 4 are so close ...

2 looks like Autum

**Reply by ebullient** [reply]
yes.. if you look @ the slides.. you can see the difference in the rest of the palette. It is a bit more than just the logo.

**Comment by maxandersen** [comment]
2 as i see it as the light for a sunrise starting a new day :)

**Reply by ebullient** [reply]
Max.. you didn't rank your other choices. ;)

**Comment by sigrunixia** [comment]
```
3 🚀. It looks the most stained glass-ish. 
4. I find the color of green more pleasing than 1.
1. The green of the tree feels muted.
2. 👎 I actually do not like this and think from a contrast standpoint, we'll run into an issue. 
```

**Comment by cowtowncoder** [comment]
* 4 (🚀)
* 1, 3
* 2

**Comment by ebullient** [comment]
4 is sneaky, because there is a hidden option (the sticker could have that dark green in place of the black, and I'm unsure about that). In most of the palettes, dark mode is actually not black: it's brown, or deep deep green. Pretty subtle stuff.

I am a fan of the pop of yellow over orange (I am truly a sucker for warm yellows), but I like the overall combination of green + blue + orange in 4 over the color tones in 2 (more olive-y greens).

So I would go: 4 (pop of orange+lightblue, and I like these greens), 1 (two variants of blue, not sure how much I'd use the brown), 2 (I love this yellow, though I would love a complementary blue in the mix, I would wish for less-olive-y greens, but the set works), then 3 (these greens are just not my cup of coffee).

**Comment by cealsair** [comment]
Here are my preferences from highest to lowest:
4
3
1
2

**Comment by evanchooly** [comment]
3 4 1 2

**Reply by Sanne** [reply]
Same for me :)
3 ranks best because it's colorful and not in a sad season.

**Comment by yrodiere** [comment]
4, 1.

They look good and the palettes offer us three distinct colors (not counting shades of green or black) to work with.

:-1: 2, 3: these basically offer us only two distinct colors. IMO such limitations are going to be very annoying when trying to build on the palette, e.g. for slide decks.

**Comment by nmcl** [comment]
4, 2, 3, 1

**Comment by gavinking** [comment]
My wife and I vote for 3.

**Reply by ebullient** [reply]
Man. You and Max. What order for the rest?! =)

**Reply by gavinking** [reply]
- 3 🚀
- 4 (colors a bit washed out and house does not stand out as much)
- 1 (boring camo color scheme hoo yah)
- 2 (looks like the white balance was way off)

**Comment by kenfinnigan** [comment]
- 4 🚀
- 3
- 1
- 2

[a comment by an automated account is not shown]

**Reply by maxandersen** [reply]
Educate me what magic potion did I not drink to get skipped as voting?

**Reply by ebullient** [reply]
You did not get skipped, though I will take the feedback that the output for comment counting should list at least the number of other respondents, if not their ids.

Results for votes based on reactions also only list the members of the required group, but you do at least see where the other votes landed.

You are not (yet) in the founders group. I can add you, if that is your wish.

The point of this kind of vote counting is to make sure that we have votes from the required group of people (though others are nice and their input does matter). My aim here is to facilitate being able to make decisions asynchronously, and that means knowing that all of the required votes are present and accounted for. ;)

**Comment by ebullient** [comment]
Fantastic! Thanks to everyone for responding. Onwards! 🍻
